harvester_e2e_tests/apis/test_settings.py

```diff
@@ -162,3 +162,35 @@ def test_invalid_S3(self, api_client):
             f"S3 backup-target should check key/secret/bucket/region"
             f"API Status({code}): {data}"
         )
+
+
+@pytest.mark.p0
+@pytest.mark.negative
+@pytest.mark.settings
+@pytest.mark.settings
+@pytest.mark.skip_version_before('v1.3.1')
+class TestUpdateInvalidKubeconfigTTLMinuteSetting:
+    def test_invalid_kubeconfig_ttl_min(self, api_client):
+        KubeconfigTTLMinSpec = api_client.settings.KubeconfigDefaultTokenTTLSpec.TTL
+        spec = KubeconfigTTLMinSpec(99999999999999)
+        code, data = api_client.settings.update('kubeconfig-default-token-ttl-minutes', spec)
+        assert 422 == code, (
+            f"Kubeconfig Default Token TTL Minutes should not exceed 100yrs\n"
+            f"API Status({code}): {data}"
+        )
+
+
+@pytest.mark.p0
+@pytest.mark.settings
+@pytest.mark.settings
+@pytest.mark.skip_version_before('v1.3.1')
+class TestUpdateValidKubeconfigTTLMinuteSetting:
+    def test_valid_kubeconfig_ttl_min(self, api_client):
+        KubeconfigTTLMinSpec = api_client.settings.KubeconfigDefaultTokenTTLSpec.TTL
+        spec = KubeconfigTTLMinSpec(172800)
+        code, data = api_client.settings.update('kubeconfig-default-token-ttl-minutes', spec)
+        print("here")
```

Comment:
Please remove temporary code.

Reply:
Thanks for that mention :smile: :+1: - I've cleaned up that debugging code in the latest push